lib/Swagger/src/Discussion/Api/LeaderboardsApi.php

```diff
@@ -0,0 +1,315 @@
+                case 400:
+                    $data = $this->apiClient->getSerializer()->deserialize($e->getResponseBody(), '\Swagger\Client\Discussion\Models\HalProblem', $e->getResponseHeaders());
+                    $e->setResponseObject($data);
+                    break;
+            }
+
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation getReportsLeaderboard
+     *
+     * Gets the reports leaderboard for a site
+     *
+     * @param int $site_id The id of the site (required)
+     * @param int $days The number of days to generate the leaderboard for (optional, default to 30)
+     * @return \Swagger\Client\Discussion\Models\LeaderboardResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getReportsLeaderboard($site_id, $days = null)
+    {
+        list($response) = $this->getReportsLeaderboardWithHttpInfo($site_id, $days);
+        return $response;
+    }
+
+    /**
+     * Operation getReportsLeaderboardWithHttpInfo
+     *
+     * Gets the reports leaderboard for a site
+     *
+     * @param int $site_id The id of the site (required)
+     * @param int $days The number of days to generate the leaderboard for (optional, default to 30)
+     * @return Array of \Swagger\Client\Discussion\Models\LeaderboardResponse, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getReportsLeaderboardWithHttpInfo($site_id, $days = null)
+    {
+        // verify the required parameter 'site_id' is set
+        if ($site_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $site_id when calling getReportsLeaderboard');
+        }
+        // parse inputs
+        $resourcePath = "/{siteId}/leaderboard/reports";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/hal+json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array('application/json'));
+
+        // query params
+        if ($days !== null) {
+            $queryParams['days'] = $this->apiClient->getSerializer()->toQueryValue($days);
+        }
+        // path params
+        if ($site_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "siteId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($site_id),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)